### PR TITLE
@task PDA-1102 Display the fixed box at the bottom for an open order’s details view

### PR DESCRIPTION
**Pull request by tren-parasoft**
Display the fixed box at the bottom for an open order’s details view.

### CODE REVIEW

**Comment by yshi-parasoft** [comment]
Please merge changes from main branch and I'll review after merge.

**Comment by yshi-parasoft** [comment]
image
When I try to type something inside "comment" field, the opened keyboard will cover the "comment" field. 
This should be handled correctly. Please refer to other app in your phone for the correct behavior. Maybe the whole screen should move above the keyboard.

File: app/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -43,4 +43,9 @@
     <string name="invoice_number">任务ID:</string>
     <string name="purchase_order">员工编号:</string>
     <string name="location_loading_error">位置加载失败</string>
+    <string name="Comments_field">评论</string>
+    <string name="order_response">订单回应</string>
+    <string name="order_response_approve">通过</string>
+    <string name="order_response_deny">拒绝</string>
+    <string name="select_order_response_error">请选择订单回应结果</string>
```
Comment (yshi-parasoft) [inline app/src/main/res/values-zh/strings.xml:50]: Is this used anywhere?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -45,4 +45,9 @@
     <string name="invoice_number">Invoice number:</string>
     <string name="purchase_order">Purchase order:</string>
     <string name="location_loading_error">Error loading location</string>
+    <string name="Comments_field">Comments</string>
+    <string name="order_response">Order response</string>
```
Comment (yshi-parasoft) [inline app/src/main/res/values/strings.xml:49]: Please add "..." after this

**Comment by tren-parasoft** [comment]
For Android API > 30 : this dialog working well.
For Android API <=30 : keyboard will covers the editText.

Reason: For Android API > 30, System allow dialog out of screen by default, so the dialog will move up and the keyboard will not covers the eitText. For Android API <=30, System didn't allow dialog out of screen , and the dialog height is 95% height of the screen, so there is no space for keyboard. The editText covered by the keyboard.

**Pull request merged**